docs/docs/development/local-toolchain/setup/container.mdx

```diff
@@ -0,0 +1,273 @@
+        {label: 'Podman', value: 'podman'}
+      ]}>
+  <TabItem value="vsCode">
+    :::note
+    This approach is documented for
+    [VS Code](https://github.com/microsoft/vscode) not
+    [Code OSS](https://github.com/microsoft/vscode/wiki/Differences-between-the-repository-and-Visual-Studio-Code).
+    :::
+
+    1. Install [Docker Desktop](https://www.docker.com/products/docker-desktop)
+       for your operating system.
+    2. Install [VS Code](https://code.visualstudio.com/).
+    3. Install the [Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers).
+
+  </TabItem>
+  <TabItem value="cli">
+    1. Install [Docker Desktop](https://www.docker.com/products/docker-desktop)
+       for your operating system.
+    2. Install the [Dev Container CLI](https://github.com/devcontainers/cli).
+  </TabItem>
+  <TabItem value="podman">
+    1. Install Podman for your operating system:
+        - [Podman CLI](https://podman.io/docs/installation)
+        - [Podman Desktop](https://podman-desktop.io/docs/installation)
+
+    :::warning
+    Please follow the installation instructions carefully. If you do not have
+    an working **Podman machine**, executing the commands below won't be
+    possible.
+    :::
+
+  </TabItem>
+</Tabs>
+
+## Creating Volumes
+
+In case you have a `zmk-config` or want to build with additional modules, it is
+necessary to first make them available by creating volumes.
+
+<Tabs
+  groupId="container"
+  defaultValue="vsCode"
+  values={[
+    { label: "Docker", value: "vsCode" },
+    { label: "Podman", value: "podman" },
+  ]}
+>
+  <TabItem value="vsCode">
+    When setting up the container, either using
+    [VS Code](https://code.visualstudio.com/) or the
+    [Dev Container CLI](https://github.com/devcontainers/cli), the volumes will
+    automatically be mounted.
+
+    #### Zmk-Config
+    ```sh
+    docker volume create --driver local -o o=bind -o type=none \
+      -o device="/absolute/path/to/zmk-config/" zmk-config
+    ```
+
+    #### Modules
+    ```sh
+    docker volume create --driver local -o o=bind -o type=none \
+      -o device="/absolute/path/to/zmk-modules/parent/" zmk-modules
+    ```
+
+  </TabItem>
+  <TabItem value="podman">
+    #### Zmk-Config
+    ```sh
+    podman volume create --driver local -o o=bind -o type=none \
+     -o device="/absolute/path/to/zmk-config/" zmk-config
+    ```
+
+    #### Modules
+    ```sh
+    podman volume create --driver local -o o=bind -o type=none \
+      -o device="/absolute/path/to/zmk-modules/parent/" zmk-modules
+    ```
+
+  </TabItem>
+</Tabs>
+
+:::tip
+Once this is done, you can refer to the `zmk-config` with
+`/workspace/zmk-config` and `/workspace/zmk-modules` to the modules instead of
+using their full path like it is shown in the
+[build commands](../build-flash.mdx), since these are the locations they were
+mounted to in the container.
+:::
```

Comment:
```suggestion
Once this is done, you can refer to the `zmk-config` with
`/workspace/zmk-config` and `/workspace/zmk-modules` to the modules instead of
using their full path like it is shown in the
[build commands](../build-flash.mdx), since these are the locations they were
mounted to in the container.
```
Box fatigue, I think this works fine as plaintext.